content/features/featuresDeepDive/input/virtualJoysticks.md

```diff
@@ -35,3 +35,7 @@ To create a custom virtual joystick and modify it to suit specific use cases, th
 <Playground id="#C6V6UY#5" title="Custom Joystick Example" description="Simple example showing how to add a Custom Joystick to your scene." image="/img/playgroundsAndNMEs/divingDeeperVirtualJoystick2.jpg"/>
 
 As seen in this example, no overlay canvas is used so other GUI elements will continue to function and the visuals can be modified/positioned if needed.
+
+You can also create a custom input for a camera. For example, here is a joined input suitable for FPS games with a movement joystick on the left and swipe-pan area on the right:
+
+<Playground id="#MQ9B0X#2" title="Custom Joystick Input Example" description="Simple example showing how to add a Custom Joystick Input to your camera." image="/img/playgroundsAndNMEs/divingDeeperVirtualJoystick2.jpg"/>

```

Comment:
Overall, it's pretty good.  Only things that I'd recommend changing in the PG would be to override the `getSimpleName()` to return something like `joystick` so that it better reflects the type of input that it represents and to remove the `console.log` call on line 197.

Reply:
Okay! Removed console.log and assigned getSimpleName() to return joystick